worker/src/RTC/SimulcastConsumer.cpp

```diff
@@ -891,6 +892,9 @@ namespace RTC
 		// Process the packet.
 		if (this->rtpStream->ReceivePacket(packet))
 		{
+			if (this->rtpSeqManager.GetMaxOutput() == packet->GetSequenceNumber())
```

Comment:
@jmillan this is exactly as it must be, right? Just to double confirm. I think so.

Reply:
Right :+1: